client/landing/stepper/declarative-flow/onboarding.ts

```diff
@@ -188,6 +218,15 @@ const onboarding: Flow = {
 					return navigate( 'domains' );
 				}
 
+				case 'design-choices': {
+					if ( providedDependencies.destination === 'launch-big-sky' ) {
+						setCreateWithBigSky( true );
+						return navigate( 'domains' );
+					}
+					setCreateWithBigSky( false );
```

Comment:
These `setCreateWithBigSky( false )` and `setCreateWithBigSky( true )` calls are what persist the users selection to the onboarding store.

I don't love using a boolean to store this selection. Using an enum seems more generic i.e. saving `designChoicesSelection` as either `'launch-big-sky'` or `'design-picker'`. But we also store the "selected design" in the onboarding store, and the difference between "selected design" and "design choice" seems sorta confusing.

`createWithBigSky` seemed less ambiguous. 🤷

Reply:
Sounds good for now. It's not confusing.

Reply:
I suppose a boolean works here since were are do not have dependencies on the particular design choice  `launch-big-sky` or `design-picker`, we simple only want to know if they select big sky. Although saving the design choice instead of a boolean might pay dividends in the future if we start including more logic around them.